### PR TITLE
Update to latest Nova

### PR DESCRIPTION
**Pull request by coreywright**
* Switch to uuidutils from oslo_utils library
* Remove XML support

### CODE REVIEW

**Comment by coreywright** [comment]
* had to remove distribute from requirements.txt, but maybe that's limited to my local environment.
* py26 tests fail because of backwards incompatible dict comprehension in upstream nova/exception.py.
* passes all 40 tests with py27.
* used Nova 2015.1.dev793 (ie https://github.com/openstack/nova/commit/257b2542d6a255146237d0c14ec6b1aa008e1c80; git master HEAD as of now)

**Comment by roaet** [comment]
I will see what is happening shortly. Sorry for any problems.

**Comment by jmeridth** [comment]
@coreywright so you're saying you know why the py26 tests failed.  Why not fix them?  Is it because we're no longer supporting it?

**Comment by roaet** [comment]
@coreywright howdy! What is the difficulty in fixing the py26 tests?

**Comment by coreywright** [comment]
tl;dr: i was explaining why the tests are failing and why the problem is not with my pull request nor with your code.  you need to get rid of the py26 tests because upstream nor we internally run nova on py26.

nova (upstream and interally) no longer supports py26.  even without my pull request your py26 tests will fail.  the problem encountered in the py26 tests (ie dict comprehension, a py27/py3 feature) i'm intimately familiar with because i maintained an internal patch for it until we migrated off of py26 (not too long ago).  i was trying to save you the trouble of figuring out the "problem" by telling you the "problem" with py26 tests was upstream code (not wafflehaus nor wafflehaus.nova nor my pull request), though i admit in retrospect that i didn't do a good job of explaining it (a few more words would have helped ;).

**Comment by jmeridth** [comment]
@coreywright well said.  your many words are fun to read.  use them more. :smile:
